### ISSUE TITLE
Email view issue if to much attachments

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#1663]
Hi, if you got some pictures or too much attachments, it takes too much place in header of email... and it's too hard to read the email body. Look on the screen:

![email](https://cloud.githubusercontent.com/assets/5773777/16413258/377f8cc0-3d31-11e6-8ee7-3c4a19fa9510.png)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Email view issue if to much attachments](https://trello.com/c/Rxwp4duH/91-email-view-issue-if-to-much-attachments)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
